liveblog/liveblog
# [LBSD-2880] - Problem loading embeds on Classic Theme

### Purpose

- This PR resolves the issue of embeds not loading properly on Classic Theme. This target mainly Instagram and Facebook.

### What has changed

- Changed IG embed from using embedly to using the official IG API
- Modified Facebook loading script to delay to avoid clash with IG package.

### Steps to test
1. Open Editor.
2. Add several embeds from social media.
3. Go to the preview. Observe that they render as expected.


Resolves: [LBSD-2880](https://sofab.atlassian.net/browse/LBSD-2880)

[LBSD-2880]: https://sofab.atlassian.net/browse/LBSD-2880?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
